google/cloud/pubsub/internal/tracing_message_batch.cc

```diff
@@ -40,8 +40,40 @@ void TracingMessageBatch::SaveMessage(pubsub::Message m) {
   child_->SaveMessage(std::move(m));
 }
 
-// TODO(#12528): Implement functionality for Flush.
-void TracingMessageBatch::Flush() { child_->Flush(); }
+void TracingMessageBatch::Flush() {
+  using opentelemetry::trace::SpanContext;
+  using AttributesList =
+      std::vector<std::pair<opentelemetry::nostd::string_view,
+                            opentelemetry::common::AttributeValue>>;
+  int64_t message_count = message_spans_.size();
```

Comment:
s/int64_t/std::int64_t/ but I would probably just say `auto const message_count = ...`. The compiler knows the type.